packages/core/node/core-node-server-internal/src/node_service.ts

```diff
@@ -42,14 +47,10 @@ export class NodeService {
     this.log = core.logger.get('node');
   }
 
-  public async preboot(): Promise<InternalNodeServicePreboot> {
+  public async preboot({ loggingSystem }: PrebootDeps): Promise<InternalNodeServicePreboot> {
     const nodeRoles = await this.getNodeRoles();
-    this.log.info(`Kibana process configured with roles: [${nodeRoles.join(', ')}]`, {
-      service: {
-        // @ts-expect-error Field not available in ECS until 8.4
-        node: { roles: nodeRoles },
-      },
-    });
+    loggingSystem.setGlobalMeta('service.node.roles', nodeRoles);
```

Comment:
Setting this from the `node` service `preboot` means that the first few log entries will not contain the meta, but [as discussed in the issue](https://github.com/elastic/kibana/issues/135674), we think this is an okay tradeoff as the alternative would require some more complex refactoring. The meta will be in place before any plugins are started up, which is most important anyway.